packages/legend-application-studio/src/stores/project-view/ProjectViewerStore.ts

```diff
@@ -181,6 +184,7 @@ export class ProjectViewerStore {
 
     // fetch project versions
     yield flowResult(this.editorStore.sdlcState.fetchProjectVersions());
```

Comment:
Can you check with Mauricio if we still need the sdlc flow to fetch the project versions. The only place we are using this now is ProjectOverviewState. If we can just use depot flow to get the versions there?

Reply:
I think for now we can leave it for the project overview state. we can think later if we want to change it.